src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
 
+        Machine [] machines = {truck,bulldozer,excavator};
+
+        for (Machine i: machines) {
```

Comment:
For the future, do not use abbreviations
```suggestion
        for (Machine machine: machines) {
```

Reply:
OK, I understand you